### ISSUE TITLE
Enable "Search Help" Document Refreshing

### ISSUE THREAD

**Issue by scotmcp** [godotengine/godot-proposals#9625]
### Describe the project you are working on

I am documenting some code abstractions I plan on offering to the godot community.

### Describe the problem or limitation you are having in your project

When writing auto documentation within my code, and reviewing the auto generated document using "Search Help", I noticed that there isn't an easy way to "refresh" a help page. Therefore if I make any changes to the code base or it's documentation, I have to close the document page and re-open it. 

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

It would be a major QOL improvement to be able to refresh the page.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

Add a refresh menu item to the right click context menu for documents (or all script files)

### If this enhancement will not be used often, can it be worked around with a few lines of script?

No, and I feel that those of us who utilize auto documentation will likely find this enhancement used frequently.

### Is there a reason why this should be core and not an add-on in the asset library?

It's not really a plugin type of thing. This is an enhancement that I believe belongs in the core engine.

**Comment by KoBeWi** [comment]
But when you save the script, the page is already automatically refreshed 🤔

**Comment by scotmcp** [comment]
If it was auto refreshing, I wouldn't have suggested the enhancement.

Hrmm, maybe I should fill out a a bug report instead.

**Comment by KoBeWi** [comment]
https://github.com/godotengine/godot-proposals/assets/2223172/33e471fd-31db-48eb-bb2e-51867f3f7451

If it doesn't work for you then it's a bug.
Though make sure to test in the newest available version.

**Comment by scotmcp** [comment]
Well for some reason it's now working after shutting down the engine last night and starting it up again this morning when I resumed work. It was not definitely not working last night.

This is the first time I started writing auto documentation to share with the community, so I guess my lack of familiarity led me to believe what I was experiencing was behavior as written.

Sorry for wasting your time!!!
